src/test/java/org/assertj/core/test/TestFailures.java

```diff
@@ -18,8 +18,40 @@
  * @author Yvonne Wang
  * @author Francis Galiegue
  */
+
 public final class TestFailures {
 
+  /**
+   * <b>Note for developers</b> :
```

Comment:
Thanks! there is a lot of code to migrate to the new pattern.
I have a preference for catchThrowable as it allows the GIVEN WHEN THEN pattern

Reply:
I think that the two approaches have their benefits and their drawbacks.  
I can update the note with `catchThrowable()` if you wish. Tell me.

Reply:
I wish then, thanks!